presto-main/src/test/java/com/facebook/presto/execution/TestMemoryRevokingScheduler.java

```diff
@@ -221,6 +248,20 @@ private OperatorContext createContexts(SqlTask sqlTask)
         return operatorContext;
     }
 
+    private void requestMemoryRevoking(MemoryRevokingScheduler scheduler)
+            throws Exception
+    {
+        scheduler.requestMemoryRevokingIfNeeded();
+        awaitAsynchronousCallbacksRun();
+    }
+
+    private void awaitAsynchronousCallbacksRun()
```

Comment:
This is really hacky.  Can you add a counter to the `MemoryRevokingScheduler` and just verify the counter was incremented?